lib/addons/karpenter/index.ts

```diff
@@ -124,62 +136,108 @@ export class KarpenterAddOn extends HelmAddOn {
         const ttlSecondsAfterEmpty = this.options.ttlSecondsAfterEmpty || null;
         const ttlSecondsUntilExpired = this.options.ttlSecondsUntilExpired || null;
         const weight = this.options.weight || null;
+        const consol = this.options.consolidation || null;
+        const repo = this.options.repository!
```

Comment:
minor: please address github action warning ";" is missing here. 

Applies elsewhere in the file.

Reply:
addressed.